packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -94,6 +83,22 @@ export function activate(context: vscode.ExtensionContext) {
     debugConfigProvider,
     checkpointsView
   );
+
+  await checkpointService.activateRequestService();
+
+  // Load the breakpoint line info stuff right here. Need to await this, otherwise
+  // we can (and will if they're stored) get breakpoint events before the line
+  // breakpoint information is fully loaded.
+  if (!BreakpointUtil.getInstance().hasLineNumberMapping()) {
```

Comment:
When will we not go into this if-block?

Reply:
See comment about for the same thing. Fixed.